extensions/data-transfer/portability-data-transfer-apple/src/main/java/org/datatransferproject/datatransfer/apple/constants/AppleConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.datatransfer.apple.constants;
+
+/**
+ * Constants are applicable to all the verticals that Apple Supports.
+ */
+public class AppleConstants {
+
+  private AppleConstants() {}
+
+  public static final String SERVICE_ID = "Apple";
```

Comment:
[for a future PR] this looks like it's related to your upstream APIs - particularly next to these other consts. Perhaps just a rename like this?

```suggestion
  public static final String DTP_SERVICE_ID = "Apple";
```